eox-core-v-1-0/prose/edit/src/design-considerations-00.md

```diff
@@ -1,13 +1,67 @@
+
+### OpenEoX Shell
+Provides the binding mechanism that combines OpenEoX Core lifecycle information to specific product identifications.
+This combination is called an "OpenEoX statement" - a complete, standalone representation of lifecycle information for a specific product.
+The Shell forms the standalone mode in OpenEoX, enabling complete lifecycle statements that can be processed independently without external context.
+
+### OpenEoX API
+Addresses the distribution, discovery, and consumption of OpenEoX Core and Shell data across vendor ecosystems and consumer applications. The API specification defines standardized endpoints, query mechanisms, and data exchange patterns that enable automated integration.
+
+## Design Principles
+
+### Standardization and Interoperability
+The framework addresses the current industry challenge where vendors use proprietary, inconsistent formats for communicating lifecycle information. By establishing a common JSON-based format, OpenEoX enables:
+- Automated processing and integration across vendor ecosystems
+- Consistent interpretation of lifecycle stages regardless of vendor
+- Reduced operational overhead in managing multi-vendor environments
+- Enhanced accuracy in infrastructure lifecycle planning
+
+### Security-First Approach
+Recognition that End-of-Security-Support represents a critical inflection point for infrastructure security has influenced the framework design. The mandatory **end_of_security_support** field ensures that security lifecycle information is always available, supporting:
+- Proactive vulnerability management planning
+- Compliance with security frameworks requiring current support status
+- Risk assessment for critical infrastructure components
+- Automated alerting when security support approaches termination
+
+## Industry Applications
+
+### Vendor Perspectives
+The framework enables vendors to:
+- Standardize EoX communication across all product lines and channels
+- Reduce support inquiries related to lifecycle confusion
+- Demonstrate proactive customer stewardship through clear lifecycle communication
+- Support automated customer notification systems
+- Facilitate integration with partner and reseller systems
+
+### Consumer Perspectives
+Organizations consuming EoX information benefit through:
+- Automated discovery of lifecycle status across multi-vendor environments
+- Proactive planning for product refreshes and migrations
+- Enhanced compliance reporting for security and regulatory frameworks
+- Reduced operational risk from unexpected support terminations
+- Streamlined asset management and inventory tracking
+
+The OpenEoX framework ensures that organizations can adopt OpenEoX in a manner that aligns with their existing infrastructure and operational models while preserving the ability to enhance integration over time.
```

Comment:
I think we can remove all mentions of framework and replace that with OpenEoX which strengthens the "brand" and is more true to what we provide: Less a framework where some additional coding would need to bring "it" to life and more a common sense vocabulary with a few grammar rules to close the when-EoX gaps.